content/v2.0/reference/flux/stdlib/pushbullet/endpoint.md

```diff
@@ -0,0 +1,87 @@
+---
+title: pushbullet.endpoint() function
+description: >
+  The `pushbullet.endpoint()` function creates the endpoint for the Pushbullet API
+  and sends a notification of type `note`.
+menu:
+  v2_0_ref:
+    name: pushbullet.endpoint
+    parent: Pushbullet
+weight: 202
+---
+
+The `pushbullet.endpoint()` function creates the endpoint for the Pushbullet API
+and sends a notification of type `note`.
+
+_**Function type:** Output_
+
+```js
+import "pushbullet"
+
+pushbullet.endpoint(
+  url: "https://api.pushbullet.com/v2/pushes",
+  token: "",
+  mapFn: (r) => ({
```

Comment:
This should be removed 👍

Reply:
I mean, `mapFn` shouldn't be in the call to `endpoint`

Reply:
Nice catch @affo! Thanks! Fixed.